src/LaunchDarkly.ServerSdk/Files/FileDataSource.cs

```diff
@@ -98,6 +98,44 @@ private void LoadAll()
             _loadedValidData = true;
         }
 
+        private static string ReadFileContent(string path)
+        {
+            for(int i = 0; ; i++)
+            {
+                try
+                {
+                    string content = File.ReadAllText(path);
+                    return content;
+                }
+                catch(IOException e) when (IsFileLocked(e))
+                {
+                    if(i > 16)
+                    {
+                        throw;
+                    }
+                    int delay = 200 * i;

```

Comment:
I'm not sure a backoff really makes sense here, even if it is a linear rather than a geometric progression. This is not like a network request where we might need to space them out very far to avoid a lot of concurrent requests; unless the other process is doing something unusual, a file write that's not done after 200ms should take only a short time to finish, not 27 seconds.

Reply:
Switched to a constant backoff of 200ms.  The first time it retries immediately, and will satisfy most scenarios